_pages/docs/request-match-rules/mockdefinition.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: single
-title: Mockdefinition
+title: mockdefinition
```

Comment:
for the title can we keep the capital M. for title it makes sense that the first work is capitalized

Reply:
Good find; fixed